tools/glyph_script_demo.py

```
@@ -0,0 +1,179 @@
+    args = ap.parse_args()
+
+    op.OVERSHOOT = 0
+    exclude = {m.strip() for m in args.exclude.split(';') if m.strip()}
+    pk = os.path.join(args.qmk, 'keyboards', 'polykybd')
+
+    import lang_demo as _LD
+    _LD.BASE_LAYER = args.layer                # render the requested layer (default _L1)
+
+    matrices = parse_layout_matrix(os.path.join(pk, 'split72', 'keyboard.json'))
+    kcs = parse_base_layer_keycodes(os.path.join(pk, 'split72', 'keymaps', 'default', 'keymap.c'))
+    matrix_kc = dict(zip(matrices, kcs))
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)